sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/DoNotHardcodeCredentials_DefaultValues.vb

```diff
@@ -19,9 +22,36 @@ Namespace Tests.Diagnostics
             Dim myPassword1 As String = Nothing
             Dim myPassword2 As String = ""
             Dim myPassword3 As String = "   "
-            Dim myPassword4 As String = "foo" 'Noncompliant {{'password' detected in this expression, review this potentially hardcoded credential.}}
+            Dim myPassword4 As String = "foo" 'Noncompliant
 
         End Sub
+
```

Comment:
I suggest to add
Private Const Secret As String = "LoremIpsum"
and
passwordDeriveBytes = new NetworkCredential("username", Secret) // Noncompliant
to demonstrate that .IsConstant(..) is working.

Reply:
Example added in unit test